keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/config/EC2Config.kt

```diff
@@ -42,6 +46,18 @@ import org.springframework.context.annotation.Configuration
 @ConditionalOnProperty("keel.plugins.ec2.enabled")
 class EC2Config {
 
+  @Bean
+  fun clusterV1() = CLUSTER_V1
```

Comment:
This feels a bit cumbersome to maintain. What would happen if we had a single `@Bean` that is the set of all supported kinds?
```kotlin
@Bean
fun supportedKinds() = setOf(CLUSTER_V1, SECURITY_GROUP_V1, CLASSIC_LOAD_BALANCER_V1, APPLICATION_LOAD_BALANCER_V1) 
```

Reply:
Couldn't get this to work in any way that was _less_ complicated than what I'd already done